<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Monitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="console">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="command"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>No Line Ending</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Line</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Carriage Return</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Both NL &amp; CR</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>8</number>
         </property>
         <item>
          <property name="text">
           <string>Custom...</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1200 baud</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2400 baud</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4800 baud</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600 baud</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200 baud</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400 baud</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600 baud</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11520 baud</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>577</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuCall">
    <property name="title">
     <string>Call</string>
    </property>
    <widget class="QMenu" name="menuPort">
     <property name="title">
      <string>Port</string>
     </property>
     <addaction name="actionRefresh"/>
     <addaction name="separator"/>
     <addaction name="actionNoPort"/>
    </widget>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="menuPort"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionClear"/>
   </widget>
   <addaction name="menuCall"/>
   <addaction name="menuTool"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionConnect">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionNoPort">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&lt;no port&gt;</string>
   </property>
   <property name="statusTip">
    <string>noport</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
